.appveyor.yml

```diff
@@ -1,70 +0,0 @@
-skip_tags: true
-
-branches:
-  only:
-    - master
-    - /^release\/.*$/
-
-build:
-  verbosity: minimal
-
-matrix:
-  fast_finish: true
-
-environment:
-  ZEUS_HOOK_BASE:
-    secure: dcqtt6sxxBV0tDkXmPZOyz96KWmjtSz6ZqRd9mw9GXC/C4Zcwqofxt2Kh4EP7hPEFIewRRl6xkhW53HgItdt7iVmFSHuufpGoSpzse8cgg3zfE08f/u0a2EOvuRjdtoi4E/9Znrj2HY+CC+G4j2UKWzp6EcSc++qbgSMl1h6zMs=
-  CARGO_HTTP_CHECK_REVOKE: false
-  channel: stable
-  arch: x86_64
-
-configuration:
-  - Test
-  - Release
-
-for:
-  - matrix:
-      only:
-        - configuration: Test
-
-    build: off # Skip default build discovery and just run tests
-    test_script:
-      - C:\MinGW\bin\mingw32-make test-rust
-
-  - matrix:
-      only:
-        - configuration: Release
-
-    branches:
-      only:
-        - /^release\/.*$/
-
-    build_script:
-      - C:\MinGW\bin\mingw32-make release
-    after_build:
-      - zeus upload -t "application/octet-stream" -n relay-Windows-%arch%.exe .\target\release\relay.exe
-      - 7z a .\relay-pdb.zip .\target\release\relay.pdb
-      - zeus upload -t "application/octet-stream" -n relay-Windows-%arch%-pdb.zip .\relay-pdb.zip
-
-install:
-  # Push job information to Zeus
-  - npm install -g @zeus-ci/cli
-  - zeus job update --status=pending -B "%APPVEYOR_PULL_REQUEST_TITLE%" -J "%APPVEYOR_JOB_NAME%" ||
-    echo "%$APPVEYOR_REPO_BRANCH%" | findstr /V "release/">nul
-  # Install the rest
-  - appveyor DownloadFile https://win.rustup.rs/ -FileName rustup-init.exe
-  - rustup-init -yv --default-toolchain %channel% --default-host %arch%-pc-windows-msvc
-  - set PATH=%PATH%;%USERPROFILE%\.cargo\bin
-  - rustc -vV
-  - cargo -vV
-  # Check out submodules
-  - git submodule update --init --recursive
-
-on_success:
-  - zeus job update --status=passed || echo "%$APPVEYOR_REPO_BRANCH%" | findstr /V "release/">nul
-
-on_failure:
```

Comment:
on_failure does not seem to be ported, is there no equivalent on gha?

Reply:
There is: https://github.com/getsentry/onpremise/blob/21d1b6d42eee14d0de93ba77fe981f9f2a82447e/.github/workflows/test.yml#L38-L42

Reply:
Also, we no longer need this, as we do not depend on Zeus for status checks. It is only an artifact storage, so it does not matter whether we report failures.